test/integration/utils/http.js

```diff
@@ -48,6 +48,9 @@ var endpoints = {
 			getHeight(ip, port) {
 				return `http://${ip}:${port}/api/node/status`;
 			},
+			getNodeStatus(ip, port) {
```

Comment:
It is realized now through WS call in `peers.js` network scenarios.

Reply:
as discussed we are not going to use web socket status, I have made code changes to use HTTP get node status.